gnomad_qc/v4/create_release/create_release_sites_ht_genomes.py

```diff
@@ -0,0 +1,42 @@
+"""Script to create release sites HT for v4 genomes."""
+import argparse
+import logging
+
+import hail as hl
+from gnomad.resources.grch38.reference_data import dbsnp
+
+from gnomad_qc.v4.resources.annotations import get_vep
+from gnomad_qc.v4.resources.constants import CURRENT_VERSION
+
+logging.basicConfig(
+    format="%(asctime)s (%(name)s %(lineno)s): %(message)s",
+    datefmt="%m/%d/%Y %I:%M:%S %p",
+)
+logger = logging.getLogger("create_release_ht")
+logger.setLevel(logging.INFO)
+
+
+def remove_missing_vep_fields(vep_expr: hl.StructExpression) -> hl.StructExpression:
+    """
+    Remove fields from VEP 105 annotations that are missing in all rows.
```

Comment:
are some of these fields just not wanted or are they all missing?

Reply:
Gtocha, I just wanted to comment to reflect the decision accurately. I think this is okay. We could add ` or excluded in the last release` but I dont think it's necessary here. As long as the decision was covered in zenhub or the notebook I think we are good!